src/m365/context/commands/context-remove.spec.ts

```diff
@@ -0,0 +1,176 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import commands from '../commands';
+import { Logger } from '../../../cli/Logger';
+import { sinonUtil } from '../../../utils/sinonUtil';
+import Command from '../../../Command';
+import * as fs from 'fs';
+import { Cli } from '../../../cli/Cli';
+import { telemetry } from '../../../telemetry';
+const command: Command = require('./context-remove');
+
+describe(commands.REMOVE, () => {
+  let log: any[];
+  let logger: Logger;
+  let promptOptions: any;
+
+  before(() => {
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    sinon.stub(Cli, 'prompt').callsFake(async (options: any) => {
+      promptOptions = options;
+      return { continue: false };
+    });
+    promptOptions = undefined;
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      telemetry.trackEvent,
```

Comment:
Since we stubbed telemetry in before, we should restore it in after, not in afterEach. Right now, telemetry will be no longer stubbed for second and following tests.